tendrl-node-agent.service

```diff
@@ -3,11 +3,12 @@ Description= A python agent local to every managed storage node in the sds clust
 
 [Service]
 Type=simple
+Environment="HOME=/var/lib/tendrl"
```

Comment:
This directory created automatically during service start during ansible execution.
However for safer side, i will do changes in the spec file to create this directory.